temel_bilgiler/kurulumlar/metin_düzenleyiciler.md

```diff
@@ -0,0 +1,104 @@
+### Giriş
+
+Bir metin düzenleyici, ne tür bir geliştirici olduğunuzdan bağımsız olarak en çok kullanılan geliştirici aracıdır. İyi bir metin düzenleyicisi; gerçek zamanlı kod denetimi, söz dizimi vurgulama ve otomatik biçimlendirme ile daha iyi kod yazmanıza yardımcı olabilir.
+
+### Neden Microsoft Word kullanamıyorum?
+
+Microsoft Word ve Libre-Office Writer gibi zengin metin düzenleyicileri bir makale yazmak için harikadır ancak onları güzel biçimlendirilmiş belgeler oluşturmada iyi yapan özellikler onları kod yazmak için uygun kılmamaktadır. Bu zengin metin düzenleyicileriyle oluşturulan bir metin belgesi, dosyaya gömülü metinden daha fazlasına yer verir. Bu dosyalar ayrıca metnin ekranda nasıl görüntüleneceğine dair bilgiler ve belgeye gömülü grafiklerin nasıl görüntüleneceğine dair veriler içerir. Buna karşılık, VSCode ve Sublime gibi yalın metin düzenleyicileri dosyaya herhangi bir ek bilgi kaydetmez. Sadece metni kaydetmek, Ruby'nin yorumlayıcısı gibi diğer programların dosyayı kod olarak okuyup çalıştırmasını sağlar.
+
+### Kod Editörleri
+
+Kod editörlerini özelleşmiş web geliştirme araçları olarak düşünebilirsiniz. Son derece özelleştirilebilirlerdir ve hayatınızı kolaylaştıracak birçok özellik sunarlar. Programınızın neden çalışmadığını anlamak için 2 saat harcadıktan sonra süslü parantezi kapamayı atladığınızı fark etmekten daha kötü bir şey yoktur. Eklentiler, söz dizimi vurgulama, parantez ve süslü parantezlerin otomatik olarak kapatılması ve linting, bir kod editörü kullanmanın faydalarından sadece birkaçıdır. Seçebileceğiniz birçok kod editörü var ancak biz Visual Studio Code ile başlamanızı öneririz.
+
+**Visual Studio Code** ya da yaygın olarak kullanılan adıyla VSCode, mükemmel bir ücretsiz kod editörüdür. Olağanüstü eklenti desteğine ve harika bir Git entegrasyonuna sahiptir. VSCode, Odin'in öğrencileri ve moderatörleri arasında en popüler kod editörüdür. Bu nedenle toplulukta destek bulmak kolaydır.
+
+Hangi editörü kullanacağınız genellikle bir tercih meselesidir ancak bu kursun amaçları doğrultusunda VSCode kullandığınızı varsayacağız. Çünkü ücretsizdir, kullanımı kolaydır ve her işletim sisteminde hemen hemen aynı şekilde çalışır. Bunun, müfredat için VSCode dışında farklı bir metin editörü kullanıyorsanız yardım alamayacağınız anlamına geldiğini unutmayın.
```

Comment:
```suggestion
Hangi editörü kullanacağınız genellikle bir tercih meselesidir ancak bu kursun amaçları doğrultusunda VSCode kullandığınızı varsayacağız. Çünkü kullanımı kolay ve ücretsizdir, her işletim sisteminde hemen hemen aynı şekilde çalışır. Bu yüzden, müfredat için VSCode dışında farklı bir metin editörü kullanıyorsanız yardım alamayacağınız anlamına geldiğini unutmayın.
```

Reply:
Selamlar Kutay. Önerilerin için teşekkür ederim. Paragrafın sonundaki cümle önerinden pek emin olamadım. Senin önerin vesilesiyle tekrar düşündüm ve 'Aklınızda bulunsun,  müfredat için VSCode'dan farklı bir editör kullanıyorsanız, yardım alamayacaksınız.' şeklinde değiştirmeyi düşünüyorum.

Reply:
```suggestion
Hangi editörü kullanacağınız genellikle bir tercih meselesidir ancak bu kursun amaçları doğrultusunda VSCode kullandığınızı varsayacağız. Çünkü ücretsizdir, kullanımı kolaydır ve her işletim sisteminde hemen hemen aynı şekilde çalışır. Bunun, müfredat için VSCode dışında farklı bir metin editörü kullanıyorsanız yardım alamayacağınızı unutmayın.
```

`alamayacaksınız` bana biraz sert geldi. Ama dediğinden yola çıkarsak böyle bir düzeltme yapılabilir.

Reply:
Tamamdır, 'Müfredat için VSCode dışında farklı bir editör kullanıyorsanız yardım alamayacağınızı unutmayın.' şeklinde değiştiriyorum.